### DISCUSSION TITLE
Driver for new Vario (Fenix)

### DISCUSSION THREAD

**Discussion by Fenix-C** [XCSoar/XCSoar#1191] [General]
Hi,
I am looking for a developer with whom it would be possible to cooperate on the implementation of IGC Vario Fenix.

**Comment by markus2marks** [comment]
Hello, are you still looking for someone?

**Comment by Fenix-C** [comment]
Hi Markus,
Thank you for your question. By the end of the month, a driver will be
completed that will enable two-way communication with the Fenix unit. Can I
then send you new documents for consultation? I know you have a lot of work
to do, but if you would like to help us directly, I can definitely provide
you with a Fenix unit for tests.

thanks

Leoš


Dne út 24. říj 2023 9:28 uživatel Markus Marks ***@***.***>
napsal:

**Comment by markus2marks** [comment]
Yes, you can do it :)

**Comment by Fenix-C** [comment]
hi Markus,
Protocol specifications in the attachment. Is that ok?

Leoš

Dne út 24. říj 2023 19:30 uživatel Markus Marks ***@***.***>
napsal:

**Reply by markus2marks** [reply]
Hi Leoš,

I haven't received anything yet, have you sent me an e-mail already?

Markus

**Comment by DanD222** [comment]
> Protocol specifications in the attachment

Is it possible to make the protocol specifications publicly available? (see discussion in https://github.com/XCSoar/XCSoar/pull/449)

**Reply by MaxKellermann** [reply]
Agree. A vendor who creates an artificial scarcity for their technical documentation (by only emailing it privately instead of just putting it on the website for anymous download) is hostile towards their customers, and such hostility will not be rewarded by XCSoar. I mean ... what does @Fenix-C gain by keeping the documentation secret?

**Comment by Fenix-C** [comment]
hi Max,
Of course, the documentation is not secret and will be on the website soon.
However, based on the communication with Markus, we first want to check if
everything is ok for XCsoar and nothing prevents full cooperation with the
unit. I consider it stupid to post a bad thing, so I make sure of
compatibility that will be seamless and with as little effort as possible
for everyone. Unfortunately, I wrote the e-mail not only to Markus, but to
all of you 🤣

sorry
Leoš

Dne čt 2. lis 2023 18:31 uživatel Max Kellermann ***@***.***>
napsal:
